Panic on quitting devhub instead of graceful shutdown

When quitting `devhub`, the application panics with the error `panic: close of closed channel` instead of shutting down gracefully.

**Steps to reproduce:**
1. Start `devhub`
2. Quit the application (e.g., via Ctrl+C or exit command)
3. Observe the panic in the console

**Expected behavior:**
`devhub` should shut down gracefully without panicking, properly closing all resources and channels.

**Actual behavior:**
Panic occurs:
```
panic: close of closed channel

goroutine 51 [running]:
github.com/iota-uz/iota-sdk/pkg/devhub.(*StateManager).Stop(...)
...
```

**Environment:**
- OS: MacOS
- Go version: go version go1.24.2 darwin/arm64
- iota-sdk version: latest staging

**Additional context:**
See screenshot for full stack trace.

--- a/modules/warehouse/domain/entities/inventory/inventory_errors.go
+++ b/modules/warehouse/domain/entities/inventory/inventory_errors.go
@@ -1,1 +1,0 @@
-package inventory

--- a/pkg/devhub/state_manager.go
+++ b/pkg/devhub/state_manager.go
@@ -29,6 +29,9 @@ type StateManager struct {
 
 	ctx    context.Context
 	cancel context.CancelFunc
+	
+	// Prevent multiple stops
+	stopOnce sync.Once
 }
 
 type StatusUpdate struct {
@@ -297,8 +300,10 @@ func (sm *StateManager) GetSystemStats() SystemStats {
 
 // Stop gracefully shuts down the state manager
 func (sm *StateManager) Stop() {
-	sm.cancel()
-	close(sm.statusUpdateCh)
-	close(sm.resourceUpdateCh)
-	close(sm.systemUpdateCh)
+	sm.stopOnce.Do(func() {
+		sm.cancel()
+		close(sm.statusUpdateCh)
+		close(sm.resourceUpdateCh)
+		close(sm.systemUpdateCh)
+	})
 }

--- a/pkg/devhub/update.go
+++ b/pkg/devhub/update.go
@@ -56,7 +56,6 @@ func (m Model) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
 func (m *Model) updateServiceView(msg tea.KeyMsg) (tea.Model, tea.Cmd) {
 	switch msg.String() {
 	case "ctrl+c", "q":
-		m.ServiceManager.Shutdown()
 		return m, tea.Quit
 
 	case "up", "k":

